subsystem-test/framework/src/main/java/org/jboss/as/subsystem/bridge/local/ScopedKernelServicesBootstrap.java

```diff
@@ -38,37 +39,68 @@ public LegacyControllerKernelServicesProxy createKernelServices(String mainSubsy
-                    Boolean.TYPE);
+            List<Object> convertedBootOps = getConvertedBootOps(bootOperations);
 
-            List<Object> convertedBootOps = new ArrayList<Object>();
-            for (int i = 0 ; i < bootOperations.size() ; i++) {
-                ModelNode node = bootOperations.get(i);
-                if (node != null) {
-                    convertedBootOps.add(objectConverter.convertModelNodeToChildCl(node));
-                }
-            }
-
-            //Convert additional Init
             Object convertedAdditionalInit = objectConverter.convertAdditionalInitializationToChildCl(additionalInit);
             Object convertedModelVersion = objectConverter.convertModelVersionToChildCl(legacyModelVersion);
             Object convertedValidateOpsFilter = objectConverter.convertValidateOperationsFilterToChildCl(validateOpsFilter);
 
+            if (!Stability.DEFAULT.equals(stability)) {
+                Method m = clazz.getMethod("create",
+                        String.class,
+                        String.class,
+                        legacyChildFirstClassLoader.loadClass(AdditionalInitialization.class.getName()),
+                        legacyChildFirstClassLoader.loadClass(ModelTestOperationValidatorFilter.class.getName()),
+                        List.class,
+                        legacyChildFirstClassLoader.loadClass(ModelVersion.class.getName()),
+                        Boolean.TYPE,
+                        String.class);
 
-            return m.invoke(null, mainSubsystemName, extensionClassName, convertedAdditionalInit, convertedValidateOpsFilter, convertedBootOps, convertedModelVersion, persistXml);
+                return m.invoke(null,
+                        mainSubsystemName,
+                        extensionClassName,
+                        convertedAdditionalInit,
+                        convertedValidateOpsFilter,
+                        convertedBootOps,
+                        convertedModelVersion,
+                        persistXml,
+                        stability.toString());
+            } else {
+                Method m = clazz.getMethod("create",
+                        String.class,
+                        String.class,
+                        legacyChildFirstClassLoader.loadClass(AdditionalInitialization.class.getName()),
+                        legacyChildFirstClassLoader.loadClass(ModelTestOperationValidatorFilter.class.getName()),
+                        List.class,
+                        legacyChildFirstClassLoader.loadClass(ModelVersion.class.getName()),
+                        Boolean.TYPE);
 
+                return m.invoke(null,
+                        mainSubsystemName,
+                        extensionClassName,
+                        convertedAdditionalInit,
+                        convertedValidateOpsFilter,
+                        convertedBootOps,
+                        convertedModelVersion,
+                        persistXml);
```

Comment:
I'll check with the latest changes I added in the wildfly-test-legacy where we are explicitely adding the Stability level, I guess it should be possible since now it will be visible to the child classloader, but I have to double check it.